jetty-ee10/jetty-ee10-tests/jetty-ee10-test-client-transports/src/test/java/org/eclipse/jetty/ee10/test/client/transport/AbstractTest.java

```diff
@@ -143,7 +146,16 @@ protected void prepareServer(Transport transport, HttpServlet servlet) throws Ex
 
     protected Server newServer()
     {
-        QueuedThreadPool serverThreads = new QueuedThreadPool();
+        jobFailures = new CopyOnWriteArrayList<>();
+        QueuedThreadPool serverThreads = new QueuedThreadPool()
+        {
+            @Override
+            protected void onJobFailure(Throwable x)
+            {
+                super.onJobFailure(x);
+                jobFailures.add(x);
+            }
+        };
```

Comment:
This is so weird and only used in 1 test at the Servlet level, and not sure why we are checking the failures of thread pool jobs?